Cargo.toml

```diff
@@ -172,6 +172,10 @@ path = "examples/2d/texture_atlas.rs"
 name = "3d_scene"
 path = "examples/3d/3d_scene.rs"
 
+[[example]]
+name = "post_processing"
+path = "examples/shader/post_processing.rs"
```

Comment:
not sure about the naming: is "post_effect_chromatic_aberration" better ?
Also, I guess "shader/" is an OK folder to put that in, but not 100% confident.

Reply:
maybe examples/shader/post_proccessing/chromatic_aberration.rs

Reply:
I would prefer to not add a new level of directories, shader should be good 👍

Reply:
I agree about having a shallow examples tree. Example programs should show techniques. This is the post processing example, there will be no need to make another, because any variation won't change the technique.

Reply:
https://github.com/bevyengine/bevy/pull/4572#issuecomment-1107702626 there's discussion about creating a folder `rendering` ; I don't have strong opinions about the location but I think it's relevant to note here.

Reply:
The location is also not correct and should be grouped with shaders ; to be noted the alphabetical order isn't respected currently in this file so I won't overthink it more.